@navikt/core/react/src/form/combobox/FilteredOptions/FilteredOptions.tsx

```diff
@@ -72,7 +72,7 @@ const FilteredOptions = () => {
               <Loader title="Søker..." />
             </div>
           )}
-          {!isLoading && filteredOptions.length === 0 && (
+          {!isLoading && filteredOptions.length === 0 && !allowNewValues && (
```

Comment:
Må nok gjøre noe i filteredOptionsContext > ariaDescribedBy-memo. Her bruker vi ID til denne div-en for å referere til den fra input-feltet. Det blir ugyldig aria om vi bare fjerner div-en.

Reply:
Aha. Jeg skal se på det nå.